NAMESPACE

```diff
@@ -0,0 +1,14 @@
+# Generated by roxygen2: do not edit by hand
+
+export(HUCTS_aggregator)
+export(HUC_aggregator)
+export(fromHUC_finder)
+export(getHUCList)
+export(init_regions)
+export(simplifyHucs)
+export(unionHUC)
+export(unionHUCSet)
+importFrom(maptools,readShapePoly)
```

Comment:
yeah, this makes sense - you are using importFrom, so I think all of these packages are fine in Imports instead of Depends.